cms/jinja2/templates/pages/statistical_article_page.html

```diff
@@ -132,63 +133,31 @@
         })
     }}
     {# fmt:on #}
+    {% if has_corrections or has_notices %}
+        {% set openText = _("Show detail") %}
+        {% set closeText = _("Close detail") %}
```

Comment:
question: given the DS component defaults are "Show detail" / "Hide detail", can we not use those?

(aside from the fact the DS strings are not marked for translation https://github.com/ONSdigital/design-system/blob/530f3ec8277554b6a214a4e7c9decf2d95e1a6ad/src/components/details-panel/_macro.njk#L2-L3)

Note that "Hide detail" is the correct string as per designs

Reply:
My worry is that the DS has lots of strings that are not marked for translation and we will have an inconsistent UI.

Would be good to understand the DS plan for i18n/l10n
cc @MebinAbraham @AdamHawtin